src/main/java/org/testng/internal/ConstructorOrMethod.java

```diff
@@ -83,4 +92,16 @@ public String toString() {
     if (m_method != null) return m_method.toString();
     else return m_constructor.toString();
   }
+
+  public String stringifyParameterTypes() {
```

Comment:
@juherr - Thanks for that link. I didn't know about it. But in this context I can't use that, because only for the first parameter `,` shouldn't be appended but for everything else it should be added.

Reply:
It is what StringJoiner is supposed to do too.